Nos/Assets/Localization/Localizable.xcstrings

```diff
@@ -6360,6 +6408,12 @@
             "state" : "translated",
             "value" : "Español"
           }
+        },
+        "ko" : {
+          "stringUnit" : {
+            "state" : "translated",
+            "value" : "스페인어"
```

Comment:
@mplorentz what are your thoughts on this? It's the featured author category, which we display as "Español" in English. I'm thinking it should always say "Español", then, no matter what language the user is using -- a clear indication that this category is not "Spanish people" or "the Spanish language via your native language", but just "accounts in the Spanish language". If that's the approach, this should not be in an `.xcstrings` file at all, and should just be hard-coded into the app.

Reply:
I could go either way to be honest. Here in the US we are generally familiar with the Spanish word for Spanish. I imagine in many places that isn't the case. So not translating it would probably make it really unintelligible for some people. But those people also presumably wouldn't want to follow anyone in the category, so it's probably fine? Idk it feels a little bad. I also suspect most other languages don't have the same English problem where Spanish refers to the language and the people/culture. 

In any case I think we shouldn't hold up the rest of these translations for this improvement, so I'm going to approve and merge this PR.